lms/lib/courseware_search/test/test_lms_filter_generator.py

```diff
@@ -114,6 +124,88 @@ def add_user_to_cohort_group(self):
         self.courses[0].save()
         modulestore().update_item(self.courses[0], self.user.id)
 
+    def add_splittest_with_content_groups(self, groups=None):
```

Comment:
I would just call this ``add_split_test``. A split test doesn't have groups (especially not content groups).